src/components/NewBeer.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+
+export default function NewBeer() {
+  return (
+    <form method='POST' action='https://ih-beers-api2.herokuapp.com/beers/new'>
+      <label htmlFor='name'>Name</label>
+      <input type="text" id="name" name="name"/><br/>
+
+      <label htmlFor='tagline'>Tagline</label>
+      <input type="text" id="tagline" name="tagline"/><br/>
+
+      <label htmlFor='description'>Description</label>
+      <textarea id="description" rows="5" cols="50" name="description"/><br/>
+
+      <label htmlFor='firstBrewed'>First Brewed</label>
+      <input type="text" id="'firstBrewed" name="first_brewed"/><br/>
+
+      <label htmlFor='brewerTips'>Brewer Tips</label>
+      <input type="text" id="brewerTips" name="brewers_tips"/><br/>
+
+      <label htmlFor='attenuationLevel'>Attenuation Level</label>
+      <input type="number" id="attenuationLevel" name="attenuation_level"/><br/>
+
+      <label htmlFor='contributedBy'>Contributed By</label>
+      <input type="text" id="contributedBy" name="contributed_by"/><br/>
+
+      <button type='submit'>Create Beer</button>
+    </form>
+  )
+} 
```

Comment:
With react it's not exactly right approach for forms. You must use state to handle variables, also task asks you to make axios call to create new beer.

I don't know if you tested your code, but your 'action' takes me to API page, and I leave my app on localhost -> which is wrong..

Look at the code in link -> you can have form like yours, but you still need to handle input via state.
[FORM](https://github.com/Joanneseiler/lab-react-ironbeers/blob/master/src/components/AddBeerForm.js)
[HANDLE FORM](https://github.com/Joanneseiler/lab-react-ironbeers/blob/master/src/components/NewBeer.js)